resource/sass/themes/blue/variables.scss

```diff
@@ -0,0 +1,10 @@
+//page
+$default-color-page-border: red;
```

Comment:
I did not see this variable was declared in <root>/components/variables.scss??
The same for other variables